content/doc/book/managing/security.adoc

```diff
@@ -279,28 +279,28 @@ link:https://owasp.org/www-community/attacks/csrf[on the OWASP website].
 
 == Agent/Master Access Control
 
-Conceptually, the Jenkins master and agents can be thought of as a cohesive
+Conceptually, the Jenkins controller and agents can be thought of as a cohesive
 system which happens to execute across multiple discrete processes and
-machines. This allows an agent to ask the master process for information
+machines. This allows an agent to ask the controller process for information
 available to it, for example, the contents of files, etc.
 
 For larger or mature Jenkins environments where a Jenkins administrator might
-enable agents provided by other teams or organizations, a flat agent/master
+enable agents provided by other teams or organizations, a flat agent/controller
 trust model is insufficient.
 
 The Agent/Master Access Control system was introduced
 footnote:[Starting with 1.587, and 1.580.1, releases]
 to allow Jenkins administrators to add more granular access control definitions
-between the Jenkins master and the connected agents.
+between the Jenkins controller and the connected agents.
 
-image::managing/configure-global-security-enable-agent-master.png["Configure Global Security - Enable Agent => Master Access Control", role=center]
+image::managing/configure-global-security-enable-agent-controller.png["Configure Global Security - Enable Agent => Master Access Control", role=center]
```

Comment:
This will show a broken image on the page because the image has not been renamed.  Since the image file name is not shown to users, we don't need to rename the image.  It is enough to undo this change.
```suggestion
image::managing/configure-global-security-enable-agent-master.png["Configure Global Security - Enable Agent => Master Access Control", role=center]
```